File: Assignemnt-18/TASK-4/Task1.py
"""
Task 4: Data Structures with Functions
Translates a JavaScript function to Python for printing student names.
"""


def print_students(students):
    """
    Print each student name from a list.
    
    Args:
        students (list): A list of student names (strings).
    """
    print("Student List:")
    for student in students:
        print(student)


if __name__ == "__main__":
    # Test with sample student names
    sample_students = ["Alice", "Bob", "Charlie"]
    print_students(sample_students)

File: Assignemnt-18/TASK-4/Task1.js
/**
 * Prints each student name from an array
 * @param {string[]} students - Array of student names
 */
function printStudents(students) {
    console.log("Student List:");
    students.forEach((name) => {
        console.log(name);
    });
}

// Test with sample data
const studentNames = ["Alice", "Bob", "Charlie"];
printStudents(studentNames);